_config.yml

```diff
@@ -89,10 +89,27 @@ defaults:
   - scope:
       type: nodes
     values:
-      layout: node
+      sheafy: section
       excerpt: ""
   - scope:
       type: lectures
     values:
-      layout: node
+      sheafy: section
       excerpt: ""
+
+sheafy:
+  taxa:
+    section:
+      layout: sheafy/tree/default
+      sublayout: sheafy/node/section
+      clicker: section
+    theorem:
```

Comment:
Btw the thing I had trouble with when trying to do this myself was that I needed the `Definition foo. ` to get spliced into the first paragraph of the body of the node, but instead it is is broken off by a paragraph boundary. I didn't quite understand why this wasn't an issue when the node is being rendered within a section, but I figured you'd know what's going on.

Reply:
Yes, it's possible. I tried and opted for the current solution just because it's a bit simpler. I'll take care of it now.
Btw -- you still want the sections to have an enlarged heading when they're rendered as roots, right?

Reply:
!! cool! And yes, I think TOC only needs to appear on sections.

Thanks for being so patient with me! Your work is excellent and I'm very excited by it.

Reply:
@jonsterling done with the layouts!

I also replaced the broken `topmost` attribute with the `subroot` one which contains the root of the subtree being rendered in context (needed to reliably render only its TOC in the sublayout).